module/VuFind/src/VuFind/ILS/Driver/MultiBackend.php

```diff
@@ -109,6 +109,40 @@ class MultiBackend extends AbstractBase implements \Laminas\Log\LoggerAwareInter
      */
     protected $driverManager;
 
+    /**
+     * An array of methods that should determine source from a specific parameter
+     * field
+     *
+     * @var array
+     */
+    protected $sourceCheckFields = [
+        'cancelHolds' => 'cat_username',
+        'cancelILLRequests' => 'cat_username',
+        'cancelStorageRetrievalRequests' => 'cat_username',
+        'changePassword' => 'cat_username',
+        'getCancelHoldDetails' => 'cat_username',
+        'getCancelILLRequestDetails' => 'cat_username',
+        'getCancelStorageRetrievalRequestDetails' => 'cat_username',
+        'getMyFines' => 'cat_username',
+        'getMyProfile' => 'cat_username',
+        'getMyTransactionHistory' => 'cat_username',
+        'getMyTransactions' => 'cat_username',
+        'renewMyItems' => 'cat_username',
+    ];
+
+    protected $alwaysSupportedMethods = [
```

Comment:
This property should have a `@var` comment above it. I'm also not sure I understand the values here -- not all of these methods are guaranteed to exist in all drivers (i.e. they're not part of AbstractBase or DriverInterface) so I'm not sure why they're listed as "always supported." Perhaps I misunderstand the context!